addons/editor-theme3/addon.json

```diff
@@ -364,6 +371,36 @@
       "id": "forums",
       "type": "boolean",
       "default": false
+    },
+    {
+      "name": "Insertion marker style",
+      "id": "markerStyle",
+      "type": "select",
+      "default": "fill",
+      "potentialValues": [
+        { "id": "none", "name": "No color" },
+        { "id": "fill", "name": "Fill color" },
+        { "id": "stroke", "name": "Outline color" },
+        { "id": "both", "name": "Fill and outline color" }
+      ]
+    },
+    {
+      "name": "Insertion marker opacity",
+      "id": "fillOpacity",
+      "type": "integer",
+      "min": 0,
+      "max": 100,
+      "default": 20,
+      "if": { "settings": { "markerStyle": ["none", "fill", "both"] } }
+    },
+    {
+      "name": "Insertion marker outline opacity",
+      "id": "strokeOpacity",
+      "type": "integer",
+      "min": 0,
+      "max": 100,
+      "default": 80,
```

Comment:
Could this be lower - maybe 50? The Black preset currently makes insertion markers quite similar to regular blocks.

Reply:
That sounds like a good idea

Reply:
The black preset sets the opacity to 5 though

Reply:
I assume he was talking about the outline, not the fill

Reply:
Commit https://github.com/ScratchAddons/ScratchAddons/pull/7509/commits/a1c730064b6b9cf586a795aea96896b895863ac7 changed the Black preset, but not the default value of the setting.

Reply:
In the latest commit I changed the default setting to 0 so it would look the same as vanilla Scratch by default since there is now support for a grey outline.